core/arch/arm/plat-imx/conf.mk

```diff
@@ -256,6 +286,16 @@ CFG_DDR_SIZE ?= 0x80000000
 CFG_UART_BASE ?= UART2_BASE
 endif
 
+ifneq (,$(filter $(PLATFORM_FLAVOR),imx8mnevk))
+CFG_DDR_SIZE ?= 0x80000000
+CFG_UART_BASE ?= UART2_BASE
+endif
+
+ifneq (,$(filter $(PLATFORM_FLAVOR),imx8qxpmek imx8qxplpddr4arm2 imx8qmmek imx8qmlpddr4arm2))
+CFG_DDR_SIZE ?= 0x80000000
```

Comment:
2GB is not correct. And please drop arm2 board.

Reply:
I will drop arm2.
What do you mean 2GB is not correct?

Reply:
i.MX8QXP MEK has 3GB DRAM, i.MX8QM MEK has 6GB DRAM. I did not check CFG_DDR_SIZE, would that reflect real DRAM SIZE on board? @jforissier

Reply:
`CFG_DDR_SIZE` is platform-specific it seems.
```
$ git grep -l CFG_DDR_SIZE
core/arch/arm/plat-imx/conf.mk
core/arch/arm/plat-imx/drivers/tzc380.c
core/arch/arm/plat-imx/platform_config.h
core/arch/arm/plat-stm/conf.mk
core/arch/arm/plat-stm/platform_config.h
```

Reply:
@clementfaure Is CFG_DDR_SIZE really needed for i.MX8QXP and QM?

Reply:
On MX8Q, CFG_DDR_SIZE is not needed. However, the current code requires to define CFG_DDR_SIZE for each platform:
https://github.com/OP-TEE/optee_os/blob/2f6dffbd324d3a7a15da684df5b250461d109cb5/core/arch/arm/plat-imx/platform_config.h#L37-L39